### PR TITLE
Colorable Keyhole Sweaters

### PR DESCRIPTION
**Pull request by AnywayFarus**
## About The Pull Request

In general, this is the same sweater, only with the ability to recolor it.
image
image

## How This Contributes To The Skyrat Roleplay Experience

More style, more variety, cool to see characters in different clothes, yeah!

## Changelog

:cl: Vishenka0704
add: Greyscale keyhole sweater
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/under/sweaters.dm [hunk 1]
```diff
@@ -22,7 +22,8 @@
...
 	desc = "What is the point of this, anyway?"
 	icon_state = "keyholesweater"
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
-	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/uniform_digi.dmi'
-	can_adjust = FALSE
+	greyscale_config = /datum/greyscale_config/sweater/keyhole
+	greyscale_config_worn = /datum/greyscale_config/sweater/keyhole/worn
+	greyscale_colors = "#920092"
+	can_adjust = TRUE
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/clothing/under/sweaters.dm:28]: what does this adjust to?

Comment (AnywayFarus) [reply modular_skyrat/modules/customization/modules/clothing/under/sweaters.dm:28]: @tf-4, yep, to nothing, i change it on my local rep, but now have another problem which i can't resolve.
on only #FF0000, #0000FF, cloth breaks.

Comment (AnywayFarus) [reply modular_skyrat/modules/customization/modules/clothing/under/sweaters.dm:28]: image
image

**Comment by AnywayFarus** [comment]
no idea how i can find the solution

**Pull request closed without merging**